tools/springboot/springboot_pkg.sh

```diff
@@ -34,16 +41,16 @@ COVERAGE=1
 # "bazel-out/darwin-fastbuild/bin/projects/services/basic-rest-service/coverage_runtime_classpath/projects/services/basic-rest-service_app/runtime-classpath.txt"
 # This is a workaround to ensure that the MANIFEST is picked correctly.
 if [[ $MANIFEST = *"MANIFEST.MF" ]]; then
-    GITPROPSFILE=$8
-    CLASSPATH_INDEX=$9
+    GITPROPSFILE=$9
+    CLASSPATH_INDEX=$10
+    FIRST_JAR_ARG=11
     COVERAGE=0
-    FIRST_JAR_ARG=10
 else
     # move these args down one slot, the code cov introduced something in the manifest slot
-    MANIFEST=$8
-    GITPROPSFILE=$9
-    CLASSPATH_INDEX=${10}
-    FIRST_JAR_ARG=11
+    MANIFEST=$9
+    GITPROPSFILE=$10
```

Comment:
this needs a curly brace ${10} since it is a two digit number

Reply:
Wow you are the bash magician!